Patch for KnowDe by Õsi Róbert (7szunyu@nordtelekom.hu).
This patch is for make KnowDe installable with GoboLinux's
Compile script. (version 1.8.0-2008-05-30)
===========================================================
diff -Naur knowde/Makefile knowde.new/Makefile
--- knowde/Makefile	2004-07-17 23:53:05.000000000 +0200
+++ knowde.new/Makefile	2008-05-30 00:26:02.543052591 +0200
@@ -1,7 +1,9 @@
+PREFIX=/usr/local
 HEADERS=`pkg-config --cflags gtk+-2.0` `xml2-config --cflags`
 LIBS=`pkg-config --libs gtk+-2.0` `xml2-config --libs`
 OBJS=obj/main.o obj/Knowde.o obj/KnowdeFile.o obj/MainWindow.o obj/ListView.o obj/TreeView.o obj/Menu.o obj/Editor.o obj/Toolbar.o obj/Reader.o obj/Statusbar.o obj/Searcher.o obj/DatePoint.o obj/Dater.o obj/Abouter.o obj/TimeView.o obj/ExportHTML.o obj/Layouter.o
-CC=g++ -march=i686 -mcpu=i686 -O4 -Wall
+CFLAGS=-march=i686 -O4 -Wall -fomit-frame-pointer
+CC=g++ $(CFLAGS)
 
 knowde: $(OBJS)
 	$(CC) $(LIBPATH) -o bin/knowde $(OBJS) $(LIBS)
@@ -62,8 +64,8 @@
 	$(CC) -c $(HEADERS) src/Layouter.cc -o obj/Layouter.o
 
 install:
-	install -d -o root -g root -m 0755 /usr/local/bin
-	install -o root -g root -m 0755 bin/knowde /usr/local/bin
+	install -d -o root -g root -m 0755 $(PREFIX)/bin
+	install -o root -g root -m 0755 bin/knowde $(PREFIX)/bin
 
 clean:
 	rm -rf obj/*.o *~ src/*~ bin/knowde
